lib/services/livesync/livesync-service.ts

```diff
@@ -526,8 +526,8 @@ export class LiveSyncService extends EventEmitter implements IDebugLiveSyncServi
 		}
 
 		const currentWatcherInfo = this.liveSyncProcessesInfo[liveSyncData.projectDir].watcherInfo;
-
-		if (!currentWatcherInfo || currentWatcherInfo.pattern !== pattern) {
+		const areWatcherPatternsDifferent = () => _.difference(currentWatcherInfo.pattern, pattern).length || _.difference(pattern, currentWatcherInfo.pattern).length;
```

Comment:
You might use `_.xor(currentWatcherInfo.pattern, pattern).length` (not sure, should be checked).

Reply:
Was looking for this, thanks!